server/api/createReceipt_handler.py

```diff
@@ -1,46 +1,47 @@
 # import necessary libraries
 from flask import jsonify, Blueprint
-from flask import Flask, request, json ,jsonify, make_response
-from database.models import user,receipt,picture
+from flask import Flask, request, json, jsonify, make_response
+from database.models import user, receipt, picture
 import uuid
 from flask_expects_json import expects_json
 
 
-createReceipt_handler = Blueprint('createReceipt_handler',__name__)
+createReceipt_handler = Blueprint('createReceipt_handler', __name__)
 
 
 # Create a route for '/createReceipt'
-@createReceipt_handler.route('/createReceipt',methods=['POST'])
+@createReceipt_handler.route('/createReceipt', methods=['POST'])
 def createReceipt():
     # Get Receipt inputs
     try:
-        auth_token=request.headers.get("auth_token")
-        auth=user.decode_auth_token(auth_token)
-        user_id=auth['user_id']
-        id=uuid.uuid1()
-        receipt_id=str(id).replace("-", "")
-        title=request.get_json()['title']
-        amount=request.get_json()['amount']
-        category=request.get_json()['category']
-        receipt_date=request.get_json()['receipt_date']
-        create_date=request.get_json()['date_created']
-        picture_url=request.get_json()['picture_url']
-        
+        file = request.files['images']
```

Comment:
it'll grab an entire object, rather than trying to convert it to `get_json` because i think it loses all its core data when you do that 

https://kishstats.com/python/2018/03/22/flask-amazon-s3-part-3.html

^ currently `file` is only one image. you can change that on the front-end if you want to take in multiple images